content/source/docs/cloud/run/cli.html.md

```diff
@@ -86,6 +86,12 @@ If you manage your Terraform configurations in self-contained repositories, the
 
 If you use a combined repository and [specify a working directory on workspaces](../workspaces/settings.html#terraform-working-directory), you can run Terraform from either the real working directory or from the root of the combined configuration directory. In both cases, Terraform will upload the entire combined configuration directory.
 
+## Excluding files from upload
+
+When using in the CLI-driven run workflow, an archive of your configuration directory is
```

Comment:
```suggestion
CLI-driven runs upload an archive of your configuration directory
```

Reply:
Thanks, I was trying to follow/see if "run workflow" was some sort of terminology we'd decided to stick to, but I find it confusing. Grateful to simplify :)